content/tutorial-paychan.md

```diff
@@ -0,0 +1,611 @@
+                                "DestinationTag": 20170428,
+                                "Expiration": 547073182,
+                                "Flags": 0,
+                                "OwnerNode": "0000000000000000",
+                                "PreviousTxnID": "C5C70B2BCC515165B7F62ACC8126F8F8B655EB6E1D949A49B2358262BDA986B4",
+                                "PreviousTxnLgrSeq": 29451256,
+                                "PublicKey": "023693F15967AE357D0327974AD46FE3C127113B1110D6044FD41E723689F81CC6",
+                                "SettleDelay": 86400
+                            },
+                            "LedgerEntryType": "PayChannel",
+                            "LedgerIndex": "5DB01B7FFED6B67E6B0414DED11E051D2EE2B7619CE0EAA6286D67A3A4D5BDB3"
+                        }
+                    },
+                    {
+                        "ModifiedNode": {
+                            "FinalFields": {
+                                "Account": "rN7n7otQDd6FczFgLdSqtcsAUxDkw6fzRH",
+                                "Balance": "1041862844",
+                                "Flags": 0,
+                                "OwnerCount": 2,
+                                "Sequence": 42
+                            },
+                            "LedgerEntryType": "AccountRoot",
+                            "LedgerIndex": "B1CB040A17F9469BC00376EC8719535655824AD16CB5F539DD5765FEA88FDBE3",
+                            "PreviousFields": {
+                                "Balance": "942868450",
+                                "OwnerCount": 3,
+                                "Sequence": 41
+                            },
+                            "PreviousTxnID": "C5C70B2BCC515165B7F62ACC8126F8F8B655EB6E1D949A49B2358262BDA986B4",
+                            "PreviousTxnLgrSeq": 29451256
+                        }
+                    },
+                    {
+                        "ModifiedNode": {
+                            "FinalFields": {
+                                "Flags": 0,
+                                "Owner": "rN7n7otQDd6FczFgLdSqtcsAUxDkw6fzRH",
+                                "RootIndex": "E590FC40B4F24D18341569BD3702A2D4E07E7BC04D11CE63608B67979E67030C"
+                            },
+                            "LedgerEntryType": "DirectoryNode",
+                            "LedgerIndex": "E590FC40B4F24D18341569BD3702A2D4E07E7BC04D11CE63608B67979E67030C"
+                        }
+                    }
+                ],
+                "TransactionIndex": 7,
+                "TransactionResult": "tesSUCCESS"
+            },
+            "status": "success",
+            "validated": true
+        }
+    }
+
+In the transaction's metadata, look for the following:
+
+- A `DeletedNode` entry with `"LedgerEntryType": "PayChannel"`. The `LedgerIndex` field should match the Channel ID. This indicates that the channel was deleted.
```

Comment:
This is a nitpick, but is there another way to specify which `LedgerIndex` field? Some lazy readers (ahem, not me of course) might skip over the first sentence and be confused by the multiple `LedgerIndex` fields in the response. (Though if the reader made it this far, they're probably invested enough to pay sufficient attention to those details, so feel free to ignore this if it doesn't make sense.)